backend/src/main/java/org/eclipse/tractusx/puris/backend/common/edc/logic/util/EdcRequestBodyBuilder.java

```diff
@@ -338,10 +356,42 @@ public JsonNode buildDtrRegistrationBody() {
         return body;
     }
 
+    public JsonNode buildPartTypeInfoRegistrationBody(Partner partner) {
+        var body = getAssetRegistrationContext();
+        body.put("@id", getPartTypeInfoAssetId(partner));
+        var propertiesObject = MAPPER.createObjectNode();
+        body.set("properties", propertiesObject);
+        var dctTypeObject = MAPPER.createObjectNode();
+        propertiesObject.set("dct:type", dctTypeObject);
+        dctTypeObject.put("@id", CX_TAXO_NAMESPACE + "Submodel");
+        propertiesObject.put("cx-common:version", "3.0");
+        propertiesObject.put("aas-semantics:semanticId.@id", "urn:samm:io.catenax.part_type_information:1.0.0#PartTypeInformation");
```

Comment:
ah sorry, my fault. @id should be nested same as the dct:type. Please refer to [this issue](https://github.com/eclipse-tractusx/puris/issues/188) :)